packages/elements/src/components/ino-tab-bar/ino-tab-bar.tsx

```diff
@@ -48,10 +41,34 @@ export class TabBar implements ComponentInterface {
    */
   @Event() activeTabChange!: EventEmitter;
 
+  @Watch('activeTab')
+  activeTabChangedWatcher(newTabIndex: number) {
+    if (this.mdcInstance) {
+      this.mdcInstance.activateTab(newTabIndex);
+      this.updateTabsAriaSelected(newTabIndex);
+    }
+  }
+
+  private updateTabsAriaSelected(activeTabIndex: number) {
+    const tabs = Array.from(this.el.querySelectorAll('ino-tab'));
+    tabs.forEach((tab, index) => {
+      const isSelected = index === activeTabIndex;
+      const panelId = (tab as HTMLInoTabElement).panelId;
+
+      const buttonElement = tab.querySelector('button');
+
+      buttonElement.setAttribute('aria-selected', isSelected.toString());
+      if (panelId) {
+        buttonElement.setAttribute('aria-controls', panelId);
+      }
```

Comment:
These lines assume the internal structure of the `<ino-tab>` component and therefore have too much knowledge. As soon as we do some refactoring of our tab component, we don't really realize that we might be breaking things.

That's why I would propose to implement these lines as properties of the `<ino-tab>` component. They could be named like our other accessibility props (e.g. `a11ySelected` and `a11yControls`) and just set these values to the internal `button` element. We also should write a disclaimer in the comments that these properties are managed by the `ino-tab-bar` component.

This way, the `ino-tab-bar` uses only the public API of our `ino-tab`.